libctru/include/3ds/thread.h

```diff
@@ -31,6 +31,9 @@ typedef struct Thread_tag* Thread;
  * - Processor #2 is New3DS exclusive. Normal applications can create threads on this core if the exheader kernel flags bitmask has 0x2000 set.
  * - Processor #3 is New3DS exclusive. Normal applications cannot create threads on this core.
  * - Processes in the BASE memory region can always create threads on processors #2 and #3.
+ * 
+ * @note For the detached parameter to work, you need to either return from your entrypoint or call threadExit. 
+ *       svcExitThread should not be used, or you will have to call threadFree yourself.
```

Comment:
I meant the whole note. The detached parameter will only fail to work if you bypass internal management by using svcExitThread.